src/Tabs/Tab.js

```diff
@@ -0,0 +1,71 @@
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export const Tab = (props) => {
+    const { title,
+        className,
+        disabled,
+        glyph,
+        id,
+        selected,
+        onClick,
+        tabContentProps,
+        linkProps,
+        ...rest } = props;
+
+    // css classes used for tabs
+    const linkClasses = classnames(
+        className,
```

Comment:
Now that this renders both the `<li>` and the `<a>`, I think `className` should map to the `<li>` and be combined with `'fd-tabs__item'`.  I don't think we need a `linkClasses` prop simply to add classes to the `<a>` element since they could write a selector using any attributes from the `<li>`.

Reply:
done.